apps/test-suite/tests/Video.js

```diff
@@ -28,6 +29,10 @@ export function test(t, { setPortalChild, cleanupPortal }) {
       await mp4Asset.downloadAsync();
       source = { uri: mp4Asset.localUri };
 
+      const portraitAsset = Asset.fromModule(require('../asses/portrait_video.mp4'));
```

Comment:
I think you have a typo mistake in your path.  You wrote asses instead of assets.

Reply:
Thank you, fixed! I'm still unable to get the end-to-end test suite actually building, will try again.